models/profile/index.js

```diff
@@ -8,6 +8,10 @@ const profileSchema = new Schema({
 	customCardback: String,
 	bio: String,
 	lastConnectedIP: String,
+	signupIP: String,

```

Comment:
These don't need to be in the model if we're using a copy of the profile object when querying.